src/main/java/lotto/generator/LottoNumberGenerator.java

```diff
@@ -0,0 +1,44 @@
+package lotto.generator;
+
+import static lotto.constant.LottoConstant.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoNumberGenerator implements NumberGenerator {
+
+	@Override
+	public List<Integer> generate() {
+		List<Integer> randomNumbers = createRandomNumbers();
+		List<Integer> lottoNumbers = createLottoNumbers(randomNumbers);
+		Collections.sort(lottoNumbers);
+
+		return lottoNumbers;
+	}
+
+	private List<Integer> createLottoNumbers(List<Integer> randomNumbers) {
+		List<Integer> lottoNumbers = new ArrayList<>();
+		for (int i = 0; i < VALID_LOTTO_NUMBER_COUNT; i++) {
+			lottoNumbers.add(randomNumbers.get(i));
+		}
+
+		return lottoNumbers;
+	}
+
+	private List<Integer> createRandomNumbers() {
+		List<Integer> numbers = createSeedNumbers();
+		Collections.shuffle(numbers);
+
+		return numbers;
+	}
+
+	private List<Integer> createSeedNumbers() {
```

Comment:
`createSeedNumbers()`는 LottoNumber를 위한 `List<Integer>`를 만들때마다 호출되고 있군요! 조금 더 효율적으로 처리할 수 있는 방법이 있을까요?!

Reply:
앗 LottoNumber 를 반복적으로 생성할 때 드는 비용에 대해 크게 고려하지 못한 것 같습니다.
아래 코드처럼 LottoNumberGenerator 에 seedNumbers 를 필드로 선언하고, 생성자에서 최초 1번만 초기값을 구성해준다면 좀더 효율적으로 처리가 가능할 것 같습니다.
```suggestion
	  private final List<Integer> seedNumbers;
  
	  public LottoNumberGenerator() {
		  this.seedNumbers = createSeedNumbers();
	  }
```